src/components/common/Actions.vue

```diff
@@ -149,6 +144,43 @@ export default class Actions extends Mixins(FilingTemplateMixin, LegalApiMixin)
     }
     this.setIsFilingPaying(false)
   }
+
+  private get nextRoute (): string | null {
+    const nextStep = this.next()
+    if (nextStep) {
+      return nextStep.to
+    }
+    return null
+  }
+
+  private get nextButtonLabel (): string {
+    const nextStep = this.next()
+    if (nextStep) {
+      return nextStep.text.replace('\n', ' ')
+    }
+    return ''
+  }
+
+  private next () : any {
+    const currentStep: number| null = this.$router.currentRoute.meta
+      ? this.$router.currentRoute.meta.step : null
+    if (currentStep && currentStep < this.stateModel.maxStep) {
+      return this.getSteps.find(step => step.step === currentStep + 1)
+    }
+    return null
+  }
+
+  private get previousRoute (): string | null {
+    const currentStep: number| null = this.$router.currentRoute.meta
+      ? this.$router.currentRoute.meta.step : null
```

Comment:
Potentially same as above if optional chaining works. Your call!

Reply:
I tried optional chaining first. It was showing some error. Will look into it again

Reply:
It might be that it returns `undefined` not `null`? Not 100% tho, i'll have to pull the code and try as well!

Reply:
Done